hello-world/src/workflows.ts

```diff
@@ -9,6 +9,11 @@ const { greet } = proxyActivities<typeof activities>({
 
 /** A workflow that simply calls an activity */
 export async function example(name: string): Promise<string> {
-  return await greet(name);
+  const minuteP = sleep('1m');
```

Comment:
Assuming this wasn’t supposed to be committed

Reply:
thanks!